numato_relay/setup.py

```diff
@@ -0,0 +1,29 @@
+import os
+from glob import glob
+from setuptools import find_packages, setup
+
+package_name = 'numato_relay'
+
+setup(
+    name=package_name,
+    version='0.0.0',
```

Comment:
Same here too

Reply:
This is manually updated but should match the package.xml.